docs/latest/examples/using-unocss.md

```diff
@@ -0,0 +1,60 @@
+---
+description: |
+  One can use UnoCSS, an instant on-demand atomic CSS engine
+---
+
+When you initialize a project with `deno run -A -r https://fresh.deno.dev`,
+you'll end up with a `main.ts` like the following:
+
+```ts
+/// <reference no-default-lib="true" />
+/// <reference lib="dom" />
+/// <reference lib="dom.iterable" />
+/// <reference lib="dom.asynciterable" />
+/// <reference lib="deno.ns" />
+
+import { start } from "$fresh/server.ts";
+import manifest from "./fresh.gen.ts";
+
+import unocssPlugin from "$fresh/plugins/unocss.ts";
+import unocssConfig from "./uno.config.ts";
+
+await start(manifest, { plugins: [unocssPlugin(unocssConfig)] });
+```
+
+The template generates a Twind v0 project by default. If you want to use UnoCSS
```

Comment:
I think lines 6-23 are redundant (coming from `using-twind-v1.md`) and could be removed.

Reply:
Agreed, I think the docs are pretty clear.